RadarSDK/RadarSettings.m

```diff
@@ -214,9 +215,13 @@ + (RadarFeatureSettings *)featureSettings {
 
 + (void)setFeatureSettings:(RadarFeatureSettings *)featureSettings {
     if (featureSettings) {
+        //This is added as reading from NSUserdefaults is too slow for this feature flag. To be removed when throttling is done. 
```

Comment:
Think we just want this comment once (directly above line 217). Remove the one below